src/Middleware/CORS/test/testassets/TestOrigin/Program.cs

```diff
@@ -11,15 +13,10 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseUrls("http://+:9001", "http://+:9002")
-                .UseContentRoot(Directory.GetCurrentDirectory())
-                .ConfigureLogging(factory => factory.AddConsole())
-                .UseStartup<Startup>()
-                .Build();
-
-            host.Run();
+            using (var host = WebHost.CreateDefaultBuilder(args).UseStartup<Startup>().Build())
```

Comment:
Part of the reason we use `http://+` was to force Kestrel to use the local host name (i..e. `http://mymachine:...` versus `http://localhost`). I think `CreateDefaultBuilder` does not give us the same behavior. Could you confirm?

Reply:
Kestrel doesn't use machine names, it uses IPs. The difference between http://localhost and http://+ is that localhost binds to loopback (::1), and + binds to any machine IP. CreateDefaultBuilder doesn't set a default URL, Kestrel defaults to http://localhost:5000 of nothing is specified.